src/bucket/BucketManagerImpl.cpp

```diff
@@ -859,6 +864,71 @@ BucketManagerImpl::maybeSetIndex(std::shared_ptr<Bucket> b,
     }
 }
 
+medida::TimerContext
+BucketManagerImpl::getBulkLoadTimer(std::string const& label) const
+{
+    mBucketListDBQueryMeter.Mark();
+    return mApp.getMetrics()
+        .NewTimer({"bucketlistDB", "bulk", label})
```

Comment:
From what I remember, medida is dad at this and this will be too slow given how often those metrics get used: this creates a new metric (expensive) and then throws it away every time, so you need to manage your own set/vector.